### ISSUE TITLE
Add tests for a few queue issues

### ISSUE THREAD

**Issue by fwsmit** [dunst-project/dunst#1168]
Some tests should be added that can test for any regresions in the following issues:

- [ ] #1102 
- [ ] #1163
- [ ] #1166

**Comment by bynect** [comment]
How would we go about testing this? Do we need to query the current cpu usage?

**Comment by fwsmit** [comment]
No, you can just query the function that says how long Dunst should sleep. If the sleep time is to short, the cpu usage is high
